src/components/Sidebar/styles.ts

```diff
@@ -0,0 +1,29 @@
+
+import { makeStyles } from '@mui/styles';
+
+export default makeStyles(() => ({
+  sideBar: {
+    height: '100vh',
+    backgroundColor: '#3440e3',
+  },
+  imageLink: {
+    display: 'flex',
+    justifyContent: 'center',
+    padding: '10% 0',
+  },
+  image: {
+    width: '70%',
+  },
+  subHeader: {
+    backgroundColor: '#3440e3 !important',
```

Comment:
Is the use of `!important` *absolutely* unavoidable? It is generally a "last resort" tool that is the equivalent of going nuclear.

Reply:
Yes - material UI the ListSubheader component, has a default color for background and text color, that without tagging it as important I was unable to change those colors.